modules/capability/depinject.go

```diff
@@ -0,0 +1,51 @@
+package capability
+
+import (
+	"cosmossdk.io/core/appmodule"
+	"cosmossdk.io/depinject"
+	storetypes "cosmossdk.io/store/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+
+	modulev1 "github.com/cosmos/ibc-go/api/capability/module/v1"
+	"github.com/cosmos/ibc-go/modules/capability/keeper"
+)
+
+var _ depinject.OnePerModuleType = AppModule{}
+
+// IsOnePerModuleType implements the depinject.OnePerModuleType interface.
+func (AppModule) IsOnePerModuleType() {}
+
+func init() {
+	appmodule.Register(
+		&modulev1.Module{},
+		appmodule.Provide(ProvideModule),
+	)
+}
+
+type ModuleInputs struct {
+	depinject.In
+
+	Config *modulev1.Module
+	Cdc    codec.Codec
+	Key    *storetypes.KVStoreKey
+}
+
+type ModuleOutputs struct {
```

Comment:
ditto, something basic would be good. Potentially have linters crying at some point otherwise